Dockerfile

```diff
@@ -12,22 +12,55 @@ ENV CACHE_TIME=1
 ENV RATE_UNIT=1
 ENV PAGE_REFRESH=0
 
-RUN apk add --no-cache gcc musl-dev make perl gd gd-dev sqlite-libs sqlite-dev linux-headers lighttpd && \
-  wget https://humdi.net/vnstat/vnstat-latest.tar.gz && \
-  tar zxvf vnstat-latest.tar.gz && \
-  cd vnstat-*/ && \
-  ./configure --prefix=/usr --sysconfdir=/etc && \
-  make && make install && \
-  cd .. && rm -fr vnstat* && \
-  apk del gcc pkgconf gd-dev make musl-dev sqlite-dev linux-headers && \
-  addgroup -S vnstat && adduser -S -h /var/lib/vnstat -s /sbin/nologin -g vnStat -D -H -G vnstat vnstat
+RUN true \
+    && set -e \
+    && set -x \
```

Comment:
Is there's some benefit on having the set command split to two command instead of using `-ex`?

Reply:
This is just a habit of mine and has no special effect